TestAssets/TestPackages/dotnet-dependency-context-test/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.0-preview1-*",
+  "version": "1.0.0-preview2-*",
```

Comment:
This seems reasonable, I don't see why it wouldn't work either off the top of my head